server/routes/posts.js

```diff
@@ -1,18 +1,17 @@
 const express = require("express")
 const router = express.Router()
-const protected = require("../middleware/protectRoute")
-const mediaValidation = require("../middleware/mediaValidation")
-const upload = require('../config/multerConfig')
-const {
-  savePost,
-  uploadVideos,
-  searchPosts
-} = require("../controllers/posts")
+const multer = require('multer')
+const storage = multer.memoryStorage()
+const upload = multer({ storage })
+const protected = require('../middleware/protectRoute')
+const { savePost, uploadVideo } = require('../controllers/posts')
+const { search } = require('../controllers/posts')
 
 router.post('/save-post', protected, savePost)
-router.get('/search', searchPosts)
+// router.post('/upload-video', protected, upload.single('file'), uploadVideo)
```

Comment:
Why those endpoints commented?

Reply:
I have to commented it, because this part doesn't work properly: our code crushes with err.
I'll  set it to previous state